examples/smart_table_random/smart_table_random.cc

```diff
@@ -139,23 +139,23 @@ auto test_smart_table(const int & n, mt19937 & rng)
                     }
                     shuffle(random_set.begin(), random_set.end(), rng);
                     int how_many = rand1_to_n(rng);
-                    // string_rep << "x[" << index_of(copy_x[split_points[i]], x) << "] ";
-                    // string_rep << constraint_type_str(constraint_type);
-                    // string_rep << " {";
+                    if (make_string_rep) string_rep << "x[" << index_of(copy_x[split_points[i]], x) << "] ";
```

Comment:
What's wrong with an `if () {` block?

Reply:
Whoops. Find and replace... 
Will remedy